docs/en-US/adding-a-new-site/index.md

```diff
@@ -18,15 +18,15 @@ I'm going to walk through setting up a blog named vvvtest.com locally using VVV,
 
 If you're migrating a site from VVV 1, read this page, then visit the [migration page](migrating-vvv1.md) for further details.
 
-## `vvv-custom.yml` and The Main Folder
+## `vvv-custom.yml` and the Sites Folder
```

Comment:
This suggests a folder named `sites`, I'd prefer instead we used the root folder or removed it